src/applications/gi/selectors/calculator.js

```diff
@@ -451,6 +451,18 @@ const getDerivedValues = createSelector(
 
     const totalHousingAllowance = monthlyRateFinal * termLength;
 
+    let bah;
+    if (__BUILDTYPE__ !== 'production') {
+      if (inputs.beneficiaryLocationQuestion === 'no' &&
```

Comment:
The old behavior uses the `bah` from the institution endpoint. This becomes the "default" behavior - if the zip input is not shown or if there is no value from the end point (`beneficiaryLocationBah`) then the calculation defaults to using the default value.